api/src/main/java/io/strimzi/api/kafka/model/kafka/quotas/QuotasPluginKafka.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.kafka.quotas;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Buildable(
+    editableEnabled = false,
+    builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({"type", "producerByteRate", "consumerByteRate", "requestPercentage", "controllerMutationRate"})
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class QuotasPluginKafka extends QuotasPlugin {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_KAFKA = "kafka";
+
+    private Long producerByteRate;
+    private Long consumerByteRate;
+    private Integer requestPercentage;
+    private Double controllerMutationRate;
+
+    @Description("Must be `" + TYPE_KAFKA + "`")
+    @Override
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getType() {
+        return TYPE_KAFKA;
+    }
+
+    @Description("The default user quota on the maximum bytes per-second that each user can publish to a broker before " +
```

Comment:
TBH I copied the definition from the `KafkaUserQuotas`, as I thought that it will apply as well on the defaults quotas 😄 .

There is this block of documentation for the default Kafka quotas plugin (and the produce/fetch):
```
 By default, each unique client group receives a fixed quota in bytes/sec as configured by the cluster. This quota is defined on a per-broker basis. Each group of clients can publish/fetch a maximum of X bytes/sec per broker before clients are throttled. 
```

So, based on this ^^ the sentence: `Does it mean that if I set this to 1Mi/s that it will be 1Mi/s on each broker and I can maybe do on a three node cluster 3Mi/s in total` is correct. I can change it to `Applied` if that makes more sense.

Reply:
Well, you should make it clear for people to understand. The Kafka docs explain it quite clearly, so I guess you can use that as well if you want.